src/client.rs

```diff
@@ -67,8 +67,11 @@ impl DbClient for Client {
             }));
         }
 
-        if resp.rows.is_empty() {
```

Comment:
This will lost schema when rows is empty, it should at least return schema when query empty result.